packages/class-transformers/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@algoan/nestjs-class-transformers",
```

Comment:
This file is incorrect. It lacks `publishConfig` and other properties. See [`class-validators/package.json`](https://github.com/algoan/nestjs-components/blob/master/packages/class-validators/package.json) for more details.